clusterctl/README.md

```diff
@@ -0,0 +1,96 @@
+# 
+# Deploy a mgmt and a workload cluster with clusterctl in kind
+# 
+# This README is based on, read them before continuing
```

Comment:
This README is based on ?? did you miss something here?

Reply:
Ah got it

Reply:
clarified